GP3.Client/MauiProgram.cs

```diff
@@ -32,6 +32,11 @@ public static MauiApp CreateMauiApp()
         builder.Services.AddTransient<StatisticsPage>();
         builder.Services.AddTransient<StatisticsViewModel>();
 
+        builder.Services.AddSingleton<SettingsPage>();
+        builder.Services.AddSingleton<SettingsViewModel>();
+        builder.Services.AddSingleton<SettingsService>();
```

Comment:
I read somewhere that Pages and ViewModels should be transients because otherwise they might not be GC'ed (could be wrong though :D)